src/ophyd_async/core/_signal_backend.py

```diff
@@ -38,17 +38,17 @@
     | Array1D[np.float64]
     | np.ndarray
     | Sequence[str]
-    | Sequence[StrictEnum]
+    | Sequence[EnumTypes]
```

Comment:
`Sequence[StrictEnum | SubsetEnum | SupersetEnum]` would be "a sequence of different enum subclasses". I think this should be:
```suggestion
    | Sequence[StrictEnum]
    | Sequence[SubsetEnum]
    | Sequence[SupersetEnum]
```